docs/how_to_guide/plot_06_sklearn_pipeline_cv_demo.md

```diff
@@ -538,7 +529,7 @@ The plot confirms that the firing rate distribution is accurately captured by ou
 :::{warning}
 Please note that because it would lead to unexpected behavior, mixing the two ways of defining values for the parameter grid is not allowed. The following would lead to an error:
 
-```python
+```{code} ipython
```

Comment:
`.set_transformer_input(1)` should be added to all basis definitions in this code block